controllers/update_status.go

```diff
@@ -118,10 +127,12 @@ func (s UpdateStatus) Reconcile(r *FoundationDBClusterReconciler, context ctx.Co
 	status.IncorrectPods = make([]string, 0)
 	status.FailingPods = make([]string, 0)
 
-	configMap, err := GetConfigMap(context, cluster, r)
+	configMap, err := GetConfigMap(cluster)
 	if err != nil {
 		return false, err
 	}
+
+	// TODO (johscheuer): should be process specific #377
```

Comment:
If we already have an issue tracking it, do we need a TODO comment in the code?

Reply:
Actually I like a todo in the code base even if we have an issue for it this makes it easier to identify the place for the fix and I will be remembered that there is an issue when I take a look at this code path again.